src/backend/tools/index.ts

```diff
@@ -291,6 +291,83 @@ export const DXVK = {
           protonVerb: 'run'
         })
       })
+
+      logInfo('Removing DXVK DLLs', LogPrefix.DXVKInstaller)
+
+      // removing DXVK dlls
+      const lib32Path =
+        gameSettings.wineVersion.lib32 === undefined
+          ? ''
+          : gameSettings.wineVersion.lib32
+      const wineLib32Path = lib32Path.replace('~', userHome)
+      if (wineLib32Path === '') {
+        logError(
+          'invalid 32-bit library source directory! Will not remove DXVK DLLs',
+          LogPrefix.DXVKInstaller
+        )
+        return false
+      }
+      if (is64bitPrefix) {
+        const libPath =
+          gameSettings.wineVersion.lib === undefined
+            ? ''
+            : gameSettings.wineVersion.lib
+        const wineLibPath = libPath.replace('~', userHome)
+        if (wineLibPath === '') {
+          logError(
+            'invalid 64-bit library source directory! Will not remove DXVK DLLs',
+            LogPrefix.DXVKInstaller
+          )
+          return false
+        }
+        dlls32.forEach((dll) => {
+          if (!isMac) {
```

Comment:
btw this !isMac check is wrong, i will be making a PR to fix this in the rest of the file soon

Reply:
Thanks for highlighting this, just pushed a new commit to address it.